### PR TITLE
React: rename components extensions

### PR DESCRIPTION
**Pull request by williamdclt**
As result of the [issue 119](https://github.com/theodo/theodo-stack-generator/issues/119)

### CODE REVIEW

File: generators/react/templates/scripts/generators/component/style.css.hbs [hunk 1]
```diff
@@ -1,3 +1,3 @@
-.{{ dashCase name }}-container {
+.{{ dashCase name }}.wrap {
```
Comment (georgesbiaux) [inline generators/react/templates/scripts/generators/component/style.css.hbs]: `-wrap` ? It is supposed to respect the css naming convention

Comment (georgesbiaux) [inline generators/react/templates/scripts/generators/component/style.css.hbs]: `-wrap` ? It is supposed to respect the css naming convention

Comment (williamdclt) [reply generators/react/templates/scripts/generators/component/style.css.hbs]: Good catch, I've been too hasty with the search-and-replace

**Comment by williamdclt** [comment]
@georgesbiaux rebased, can you double-check? There was a lot of conflicts

**Comment by benja-M-1** [comment]
@williamdclt you must change the standard http://blog.m33.network/2017/11/react-component/ 

cc @lechinoix

**Pull request merged**